aten/src/ATen/core/Dimname.h

```diff
@@ -37,7 +37,7 @@ struct CAFFE2_API Dimname {
 
 using DimnameList = c10::ArrayRef<Dimname>;
 
-static Symbol kWildcard = Symbol::dimname("*");
+extern Symbol kWildcard;
```

Comment:
Hmm, the `extern` here looks suspicious.  I get a feeling this won't work on Windows.  If we need a wired in symbol, it might be better to just add it to the list in `aten/src/ATen/core/interned_strings.h ` (with a special case I suppose because you can't actually define it as a wildcard)

Reply:
For future reference, what's the problem with global variables on windows? Is it DLL related?

Reply:
Well, I guess it worked, cuz CI likes it. I just get very nervous about sharing symbols across DLLs in Windows, it works very differently than in Linux.